schematics/types/base.py

```diff
@@ -885,3 +885,100 @@ def validate_regex(self, value):
             if self.locale_regex is not None and self.locale_regex.match(locale) is None:
                 raise ValidationError(
                     self.messages['regex_locale'].format(locale))
+
+
+class PolyType(BaseType):
+
+    def __init__(self, type_classes, **kwargs):
+        if isinstance(type_classes, (list, tuple)):
+            self.type_classes = tuple(type_classes)
```

Comment:
The `type_classes` member (and the corresponding `__init__` parameter) should perhaps be called `type_instances` or just `types` instead, since (unlike in `PolyModelType`) the items are not really classes.